### DISCUSSION TITLE
Add Time Goals Feature

### DISCUSSION THREAD

**Discussion by Marlon154** [solidtime-io/solidtime#309] [Feature requests]
## Overview
Add the ability to set and track time-based goals (e.g., 5 hours/week for project research) to help users manage their time commitments effectively.

## Core Features
- Set time goals for projects/tasks/clients (daily/weekly/monthly)
- Progress tracking dashboard with visual indicators

Optional
- Alerts for under/over goal progress

## Benefits
- Better project time management
- Improved resource allocation
- Enhanced productivity tracking
- Structured time commitment planning

**Comment by Onatcer** [comment]
Thanks for the suggestion. We have discussed this internally already and think it would be a great feature for solidtime, but it is not a high priority for now.

I will convert this issue into a discussion as we handle feature requests in discussions and people can upvote it so we can see how many users are interested in it, and potentially prioritize it higher.

**Reply by Marlon154** [reply]
Hello,

A year has past. Did your priorities changed?

Did you already considered how to implement this?
I imagined per user time goal for billable hours. Alternatively, a per project goal would useful. There are many possible implementations of this. What is your take?

I may th able to work on this. 
Implementation advice is welcome.
